src/select-label.dust

```diff
@@ -0,0 +1,12 @@
+<div class="c-select c--label">
+    <label class="c-select__label">{label}</label>
+    <span class="c-select__value"></span>
+    <select class="js-linked-element" data-linked-element=".{linkedClass}">
```

Comment:
What's the point of the js-linked-element and data-linked-element? It doesn't appear to be used